recipe/meta.yaml

```diff
@@ -22,23 +22,17 @@ requirements:
     - make
     - libtool
   host:
-    - icu
     - libgcrypt 1.9.3
-    - libgpg-error 1.42
-    - libiconv 1.16  # [not linux]
+    - libgpg-error 1.42  # [linux]
     - libtool
```

Comment:
I'm not sure. I tend to say no, but I'll think more about it.

Reply:
In order to avoid dependency conflicts it might be a good idea to explicitly indicate the pinnings of `libtool`
    If other dependency conflicts result as a consequence, then we might consider modifying the pinings or removing them.